tests/src/Content/Text/BBCodeTest.php

```diff
@@ -259,7 +259,7 @@ public function dataBBCodes()
 				'text' => '[emoji=https://fedi.underscore.world/emoji/custom/custom/heart_nb.png]:heart_nb:[/emoji]',
 			],
 			'task-12900-multiple-paragraphs' => [
-				'expectedHTML' => '<h1>Header</h1><ul class="listbullet" style="list-style-type:circle;"><li>One</li><li>Two</li></ul><p>This is a paragraph<br>with a line feed.</p><p>Second Chapter</p>',
+				'expectedHTML' => "<h1>Header</h1><ul class=\"listbullet\" style=\"list-style-type:circle;\"><li>One</li><li>Two</li></ul><p>This is a paragraph<br>with a line feed.</p><p>Second Chapter</p>",
```

Comment:
Why the change here?

Reply:
Because there is some `\n` inside.

Reply:
And I'm of course an idiot, since I did it with the wrong string :grin: